integration/integration_suite_test.go

```diff
@@ -111,19 +113,33 @@ var _ = SynchronizedBeforeSuite(func() []byte {
 	inPath = sd.InPath
 	outPath = sd.OutPath
 
+<<<<<<< HEAD
+=======
+<<<<<<< HEAD
+<<<<<<< HEAD
+	if accessKeyID != "" {
+		Ω(accessKeyID).ShouldNot(BeEmpty(), "must specify $S3_TESTING_ACCESS_KEY_ID")
+		Ω(secretAccessKey).ShouldNot(BeEmpty(), "must specify $S3_TESTING_SECRET_ACCESS_KEY")
+=======
+>>>>>>> concourse-master
```

Comment:
This'll need to be resolved as well. I won't mark all of these, but anywhere the merge conflict notation with the <<<<, >>>>, or ==== is present, those conflicts will need to be resolved.